src/models_transfer_learning.py

```diff
@@ -1,5 +1,9 @@
 import tensorflow as tf
```

Comment:
@palonso Should we move the whole file also to the `tensorflow.compat.v1`?

Reply:
Yes, otherwise we will have problems with `disable_v2_behavior()`

Reply:
updated the call to `disable_v2_behavior()`. We can simplify later if we need to.

Reply:
there are more TF1 objects that are only available under the `tf.compat.v1.` scope (e.g., `truncated_normal_initializer`, `variable_scope`...).

The quick solution would be,
`import tensorflow as tf` → `import tf.compat.v1 as tf`

Otherwise, we have to identify all the objects that require `tf.compat.v1` and modify them as `disable_v2_behavior()`